meetup/models.py

```diff
@@ -1,29 +1,24 @@
 from django.db import models
 from pizzaplace.models import PizzaPlace
 from django.core.validators import RegexValidator
-from meetup.services.meetup_api_lookup_agent import MeetupApiLookupAgent
+from meetup.services.meetup_api import MeetupApi
 from django.core.exceptions import ValidationError
-
+from model_utils.models import TimeStampedModel
 
 def validate_urlname(link):
   validator = RegexValidator(
     regex='meetup\.com\/\w+(-\w+)*\/$',
```

Comment:
Are you sure this regex + capture rule work correctly? Is there a test around this? If you run this regex against the string `meetup.com/papers-we-love/`, 1) is it a match? 2) What value is captured?